testgrid/cmd/entomologist/main_test.go

```diff
@@ -61,19 +61,32 @@ func TestOptions(t *testing.T) {
 			name: "missing output, reject",
 			args: []string{"--github-org=testorg", "--github-repo=testrepo"},
 		},
+		{
+			name: "both one-shot and poll-time, reject",
+			args: []string{
+				"--github-org=testorg",
+				"--github-repo=testrepo",
+				"--output=gs://foo/bar",
+				"--gcs-credentials-file=/usr/foo/creds.json",
+				"--one-shot",
+				"--poll-time=1h",
+			},
+		},
 		{
 			name: "required options with gcs bucket output",
 			args: []string{
 				"--github-org=testorg",
 				"--github-repo=testrepo",
 				"--output=gs://foo/bar",
 				"--gcs-credentials-file=/usr/foo/creds.json",
+				"--poll-time=1h",
```

Comment:
Putting a poll time here implies that "poll time" is necessary to send data to GCS. It's not uncommon to look at test cases to try and figure out what code is doing.

Consider putting "custom poll time" in a different test case, and just leaving the others unmodified.

Reply:
agree, this isn't a required option; I would expect one test case verifying the default poll time, and another for an explicit non-default poll time

Reply:
I agree. I did not want to clutter the file with almost-similar test cases, but better to do things right.